### ISSUE TITLE
[Reviewer Tools] Weighting Incentive Adjustment 

### ISSUE THREAD

**Issue by erosman** [mozilla/addons-server#7865]
Currently the review incentive rewards are set as the following:

POST_REVIEW_WEIGHT_HIGHEST_RISK 151+ -> 140 points
POST_REVIEW_WEIGHT_HIGH_RISK 101-150 -> 120 points
POST_REVIEW_WEIGHT_MEDIUM_RISK 21-100 -> 90 points
POST_REVIEW_WEIGHT_LOW_RISK 0-20 -> 0 points

Specially on top, where the weighting can go as high as 350 (for complex add-ons), the incentives do not appear to match the weighting fairly.

Suggestion:

POST_REVIEW_WEIGHT_HIGH_RISK and HIGHEST_RISK ->  points equal to weight
POST_REVIEW_WEIGHT_MEDIUM_RISK 21-100 -> 100 points
POST_REVIEW_WEIGHT_LOW_RISK 0-20 -> 0 points

**Comment by diox** [comment]
@jvillalobos for consideration

**Comment by jvillalobos** [comment]
Redirecting to @caitmuenster and @wagnerand.

**Comment by caitmuenster** [comment]
I talked about this with @wagnerand and @kewisch a few days ago, and we're open to recalculating points. 

I do like @erosman's proposed updates. Andreas and Philipp, could you weigh in?

**Comment by kewisch** [comment]
I like the suggestion, thanks erosman for putting it up! Especially in relation to the current points, it keeps in mind that we don't want to encourage low reviews, and even HIGH is similar to before giving 101-150 points.

One thing we should keep in mind is the effect on incentives. Points can currently go as high as 800 in theory, so reviewing 3 of those would move someone from level 1 to 2. This might not be as much of a problem because in practice, non-admin reviews are lower.

**Comment by caitmuenster** [comment]
Thanks @kewisch! I'm not too concerned about moving levels because of the [new recognition program](https://wiki.mozilla.org/Add-ons/Reviewers/Recognition). However, you bring up a good point about proportional weights. The incentive program will compare the number of reviews and review weight, which will help balance those concerns.

**Comment by wagnerand** [comment]
I like the suggestion but wanted to make a small tweak to it to incentivize high(est) risk add-on reviews more. I realized making changes without also adapting the risk categories bares risk on our side.

I suggest that we take a look at #6930 and #6821 first, potentially adapt other weights based on what we learned since we started post review and then see whether we have to change the category thresholds (which is #7863).
